src/resources/settings.yml

```diff
@@ -1,47 +1,34 @@
 ---
 ##
- # The BSL Forward Proxy configuration file.
+ # The BSL Reverse Proxy configuration file.
  # _____________________________________________________________________________
  #
  # Created by brightSPARK Labs
  # www.brightsparklabs.com
  ##
 
-bsl_forward_proxy:
+reverse_proxy:
   # Settings pertaining to the services.
   services:
     proxy:
-      # Domain of the proxy which all services will be nested under.
-      domain: brightsparklabs.local
-
       # Whether to use the Acme Staging CA for certificates.
       enable_staging_ca: false
 
       # Whether to use internal Caddy certificates rather than Acme ones (useful for testing).
       enable_internal_tls: true
 
-      # Settings pertaining to the ACME DNS challenge type.
-      acme_dns_challenge:
-        # Whether to use ACME DNS challenge (rather than HTTP challenge) to acquire certificates.
-        enable: true
-
-        # AWS access key with permissions to manipulate Route53 records for DNS challenge.
-        access_key_id: CHANGE ME
-
-        # AWS access key with permissions to manipulate Route53 records for DNS challenge.
-        secret_access_key: CHANGE ME
-
-      storage:
-        # Uncomment to use filesystem (comment out the s3 type below).
-        type: filesystem
-
-        # NOTE: The names `accesskey` and `secretkey` are inline with docker Registry schema.
-        # Uncomment to use s3 (comment out the filesystem type above).
-        #type: s3
-        #accesskey: CHANGE ME
-        #secretkey: CHANGE ME
-        #bucket: bsl.internal.bslnet.production.aws.s3.docker-registry-images
-        #region: ap-southeast-2
```

Comment:
Put this back in. There are circumstances where we don't want the proxy available to the internet, so we want to use something like acme dns to do it instead of acme well-known